grails-app/services/org/pih/warehouse/stockTransfer/StockTransferService.groovy

```diff
@@ -114,12 +117,11 @@ class StockTransferService {
         if (!params) {
             List<StockTransferItem> pendingStockTransferItems = getPendingItems(location)
 
+            Map<List<String>, List<StockTransferItem>> groupedPendingItems = pendingStockTransferItems.groupBy { StockTransferItem item ->
+                [item.location?.id, item.inventoryItem?.id, item.originBinLocation?.id, item.product?.id]
```

Comment:
Does this work ok when originBinLocation is null? Or should we need to handle that with a special key i.e. NOLOC or DEFAULT?

Reply:
yeah, I checked that "manually" and it handles `null` as part of a key fine.